src/utils/endUserMessages.js

```diff
@@ -0,0 +1,24 @@
+export default {
```

Comment:
A lot of these messages were changed, you should check with @vickymorrison that the changed ones are correct